packages/odyssey-react/src/components/Tag/Tag.test.tsx

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React from "react";
+import { render } from "@testing-library/react";
+import Tag from "./Tag";
+
+const tagList = ['Item one', 'Item two', 'Item three'];
+
+describe("Tag", () => {
+  it("should render the tag list", () => {
+    const { getByTestId } = render(
+      <Tag tags={tagList} />
+    );
+
+    expect(getByTestId('ods-tag')).toBeInTheDocument();
+  });
+
+  it("should render the tag list with list items", () => {
+    const { getByTestId } = render(
+      <Tag tags={tagList} />
+    );
+    const tagListItems = getByTestId('ods-tag').querySelectorAll('.ods-tag');
+
+    expect(tagListItems).toHaveLength(3);
```

Comment:
we can improve this test by asserting that the tag content themselves (line 17 above) are in the DOM and in the correct order rather than asserting on the presence of three class names (`.ods-tag`)